include/odp/api/spec/packet_io.h

```diff
@@ -489,6 +489,8 @@ typedef union odp_pktout_config_opt_t {
 		 */
 		uint64_t no_packet_refs  : 1;
 
+		/** Enable packet Tx completion event */
+		uint64_t tx_completion : 1;
```

Comment:
Similar to checksum enable and "insert by default" options above, this could include "send tx completion by default" option. If "send by default" option would be set, then also a default destination queue would need to specified. I guess some application could track all / most outgoing packets, and by setting the default would not require to set it for every packet (just disable on those it would not care about completion).

Reply:
I'm currently sticking to enable packet Tx completion event only on request as in our HW generating it by default will cause huge performance degration. We can extend the spec in future when new HW supports insertion by default seamlessly ?